pkg/controller/drain.go

```diff
@@ -632,7 +632,7 @@ func (o *DrainOptions) evictPodsWithPVInternal(attemptEvict bool, pods []*corev1
 			returnCh <- err
 			continue
 		}
-		glog.V(3).Info("Detached pv for pod: ", pod.Name)
+		glog.V(3).Info("Detached pv for pod: ", pod.Name, " from node ", pod.Spec.NodeName)
```

Comment:
We need to capture the time taken for detachment of PV here. We might have to introduce a time variable for the same or reuse the one used for PV eviction logic above.